### PR TITLE
data: mustafaansarii.jason

### PR DESCRIPTION
**Pull request by mustafaansarii**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5097"></a>

### CODE REVIEW

**Review by mahfuztbt — APPROVED** [review]
LGTM

**Review by mahfuztbt — CHANGES_REQUESTED** [review]
`.jason` is not the correct extension for json file, edit your file name to `mustafaansarii.json`

**Review by shubhsharma19 — CHANGES_REQUESTED** [review]
Change the name of the file to `mustafaansarii.json`

File: data/mustafaansarii.jason [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/mustafa.nds/" }
+  ],
+  "testimonials": [
+    "Rahul kr",
+    "Sachin singh",
+    "nikhil raj",
+    "Ankit kr"
+    
+  ],
+  "avatar": "https://github.com/mustafa-nds.png",
```
Comment (shubhsharma19) [inline data/mustafaansarii.jason:29]: `avatar`

**Review by shubhsharma19 — APPROVED** [review]

File: data/mustafaansarii.jason (hunk 1 above)
[the remark this answers is not in the document]
Comment (shubhsharma19) [reply data/mustafaansarii.jason]: Looks good now 👍

**Review by shubhsharma19 — APPROVED** [review]
Looks good 👍

**Review by mahfuztbt — APPROVED** [review]

**Comment by mahfuztbt** [comment]
looks good now

**Review by kumarsonsoff3 — COMMENTED** [review]
Duplicate of #5100

**Comment by kumarsonsoff3** [comment]
Duplicate of #5100

**Pull request closed without merging**